documentation/book/assembly-metrics-prometheus.adoc

```diff
@@ -0,0 +1,29 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-metrics-setup.adoc
+[id='assembly-metrics-prometheus-{context}']
+
+= Prometheus
+
+link:https://prometheus.io/[Prometheus^] provides an open-source set of components for systems monitoring and alert notification.
+
+{ProductName} uses the link:https://github.com/coreos/prometheus-operator[CoreOS Prometheus Operator^] to run a highly available Prometheus server that is suitable for use in production environments.
```

Comment:
Are we sure it is highly available? I have some doubts about it.

Reply:
I've removed to be safe.